sonic-post-syseeprom/scripts/post-syseeprom

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python2
+
+'''
+    post-syseeprom
+    Syseeprom information gathering tool for SONiC
+    This tool will be started during the start phase of pmon container, gathering syseeprom info and write to state DB.
+    It's an one-shot task since syseeprom info are static.
+    With this tool, show syseeprom CLI will be able to get data from state DB instead of access hw or cache.
+'''
+
+try:
+    from sonic_daemon_base.daemon_base import DaemonBase
+    from sonic_daemon_base.daemon_base import Logger
+    from sonic_daemon_base import daemon_base
+    import argparse
+    from swsscommon import swsscommon
+except ImportError, e:
+    raise ImportError (str(e) + " - required module not found")
+
+PLATFORM_SPECIFIC_MODULE_NAME = 'eeprom'
+PLATFORM_SPECIFIC_CLASS_NAME = 'board'
+
+ERR_ARISTA_PLATFORM = 1
+ERR_FAILED_EEPROM = 2
+ERR_FAILED_UPDATE_DB = 3
+
+EEPROM_TABLE_NAME = 'EEPROM_INFO'
+SYSLOG_IDENTIFIER = 'post-syseeprom'
+
+# Global logger class instance
+logger = Logger(SYSLOG_IDENTIFIER)
+
+def main():
+    parser = argparse.ArgumentParser(description='post-syseeprom tool')
+    parser.add_argument("-w", help="write syseeprom info into state DB", action='store_true')
```

Comment:
>"-w" [](start = 24, length = 4)

you can also give a long name such as '--write' and make your code more readable 'args.write'

Reply:
add long arguments names.